pymc3/stats.py

```diff
@@ -478,7 +481,7 @@ def gradient(w):
                      bounds=bounds,
                      constraints=constraints)
 
-        weights = w_fuller(w['x'])
+        weights = w_fuller(np.round(w['x'], round_to))
```

Comment:
This rounding is mostly to avoid having negative weights (due to rounding error during optimization sometimes we can get very small negative numbers). I think this trick (and a default of 2 decimals) should not be a problem unless you compare a large number of models (only weights smaller than 0.005 will become 0). But I admit I did this because I could not find a nicer solution :)